source/includes/examples/_energyplus.html.md

```diff
@@ -0,0 +1,259 @@
+# EnergyPlus Example
+
+The full source code for this example can be [found on GitHub][1] so you can run it yourself on the Bonsai Platform.
+
+In this example, we'll walk you through the various statements that are part of a sample implementation of [EnergyPlus][2] on the Bonsai Platform, including the simulator and the Inkling files. This is a real-world example of how to use the Bonsai Platform for HVAC control using BCVTB and EnergyPlus.
+
+While this BRAIN is training, the Bonsai AI Engine launches the EnergyPlus simulator in the background for every episode. The *energyplus_simulator.py* then drives the simulator forward a step at a time until it finishes the episode and then relaunches it for the next episode, driving the actions into it and sending state results back to the Bonsai AI Engine.
+
+## Inkling File
+
+###### Schema
+
+```inkling
+schema SimState
+    Int32{0:10} SolarIrradiation
+end
+```
+
+The `SimState` schema defines the dictionary returned from the Python simulation's advance method to the BRAIN including:
+   Double TOut,
```

Comment:
@keenbrowne looks like there is some commented code that these are referring too. Should we delete the code and its variables?